src/main/java/frc/robot/Robot.java

```diff
@@ -1082,15 +1158,14 @@ private void executeShootingStateMachine() {
         /* Decrements storage */
         mStorage.removeBall();
 
-        // We are gonna continue shooting until 
+        // We are gonna continue shooting until
         // /* Goes to complete if storage is empty, otherwise fires again */
         // if (mStorage.isEmpty()) {
         //   mShootingState = ShootingState.SHOOTING_COMPLETE;
         // } else {
         //   mShootingState = ShootingState.PREPARE_TO_SHOOT;
         // }
```

Comment:
This code needs to be reinstated, but I guess not yet...

Reply:
Why the comment then?